app/models.py

```diff
@@ -1,5 +1,7 @@
 from django.contrib.auth.models import AbstractBaseUser
 from django.db import models
+from django.contrib.auth.models import User
```

Comment:
You don't need to use this User model. We already have a User model defined in models.py file

Reply:
Oh! my bad. I'll remove this.